rviz_common/include/rviz_common/frame_position_tracking_view_controller.hpp

```diff
@@ -32,7 +32,7 @@
 #define RVIZ_COMMON__FRAME_POSITION_TRACKING_VIEW_CONTROLLER_HPP_
 
 #include <OgreQuaternion.h>
-#include <OgreVector3.h>
+#include <Ogre.h>
```

Comment:
I'm not sure why the deprecation notice recommends including `Ogre.h`, but I think it'd be better to be narrow in our includes if possible. I _think_ the header we want is `<OgreVector.h>` ([reference](https://github.com/OGRECave/ogre/blob/v1.12.13/OgreMain/include/OgreVector3.h#L1)).

Same comment for other places where we replace OgreVector3.h with Ogre.h

Reply:
I think you're right, I'll give the changes a try.